config/settings_schema.json

```diff
@@ -324,5 +324,129 @@
         "info": "Example: https://vimeo.com/booqable"
       }
     ]
+  },
+  {
+    "name": "Cookie notice",
+    "settings": [
+      {
+        "type": "checkbox",
+        "id": "cookie_notice_activated",
+        "label": "Show cookie notice"
+      },
+      {
+        "type": "select",
+        "id": "cookie_notice_color_palette",
+        "label": "Color palette",
+        "options": [
+          {
+            "value": "set-1",
+            "label": "Color set 1"
+          },
+          {
+            "value": "set-2",
+            "label": "Color set 2"
+          },
+          {
+            "value": "set-3",
+            "label": "Color set 3"
+          },
+          {
+            "value": "set-4",
+            "label": "Color set 4"
+          },
+          {
+            "value": "set-5",
+            "label": "Color set 5"
+          }
+        ],
+        "default": "set-1"
+      },
```

Comment:
Could we use new color schemes with previews
![Arc_2024-04-24 14-46-34@2x](https://github.com/booqable/impact-theme/assets/40244261/50f5fced-19d4-4c3c-aa51-5e0841fe1810)

 instead of an old version with dropdown options?
 
![Arc_2024-04-24 14-46-11@2x](https://github.com/booqable/impact-theme/assets/40244261/4da7ca06-0431-46ed-9a7e-476bf6747683)